cloudflare/import_cloudflare_spectrum_app_test.go

```diff
@@ -0,0 +1,44 @@
+package cloudflare
```

Comment:
Perhaps rename this file to `import_cloudflare_spectrum_application_test.go` for consistency?

Reply:
:+1:

Reply:
:+1: